packages/core/src/BlockLink.js

```diff
@@ -1,12 +1,22 @@
 import styled from 'styled-components'
 import Link from './Link'
+import { mapProps } from './utils'
+import getSCMigrationRef from './utils/getSCMigrationRef'
```

Comment:
why not export all of these from utils/index?

Reply:
Updated, agreed that importing from the same utils looks better